Support building Ubuntu Noble images in CI

m-bull opened the pull request.

m-bull committed:
Support building Ubuntu Noble images in CI

m-bull committed:
Use Noble support feature branch from image-utils

m-bull committed:
Ensure correct podman package installed on Noble

m-bull committed:
Install repo2docker in a virtualenv

m-bull committed:
FIx RStudio deb URL for Noble

m-bull committed:
Set polkit rules correctly for Noble

m-bull committed:
Use temporary Noble feature branch of os-manila-mount

m-bull committed:
Base64 code vncpasswd output

Command output is no longer allowed to contain non-UTF-8
characters. B64 encoding the output from vncpasswd, and
decoding it back into the passwd file ensures that Ansible
never touches anything that is not UTF-8.

m-bull committed:
Fix linting errors

m-bull commented:
Requires:
  - https://github.com/stackhpc/ansible-role-os-manila-mount/pull/44
  - https://github.com/azimuth-cloud/ansible-collection-image-utils/pull/8

m-bull committed:
TESTING ONLY: Use feature branch of az-ops in CI

m-bull committed:
Update apt cache as a separate task to avoid hangs

m-bull commented:
Something about installing `ubuntu-desktop-minimal` breaks networking - investigating.

m-bull committed:
Switch to NetworkManager

m-bull committed:
Merge branch 'main' into feat/noble

m-bull committed:
Merge branch 'main' into feat/noble

m-bull committed:
Bump Noble baseimage to release-20260209

m-bull committed:
Bump image-utils collection to 0.6.0

m-bull committed:
Stop using facts_as_vars

INJECT_FACTS_AS_VARS will default to False

m-bull committed:
Cast desktop_enabled to bool

This prevents "Conditionals must have a boolean result." warning.

m-bull committed:
Use stackhpc.os-manila-mount v27.0.0

wtripp180901 commented on .github/workflows/build_test_images.yaml:
```diff
@@ -240,6 +240,7 @@ jobs:
       - name: Set up Azimuth environment
         uses: azimuth-cloud/azimuth-config/.github/actions/setup@devel
         with:
+          azimuth-ops-version: feat/noble
```
will need to remember to take this out

m-bull replied:
Done

m-bull committed:
Merge branch 'main' into feat/noble

m-bull committed:
Bump RStudio version to 2026.01.2-418

m-bull committed:
Merge branch 'main' into feat/noble

m-bull committed:
Bump Ubuntu Noble baseimage to release-20260518

m-bull committed:
Pass the linter

amanning9 committed:
Add aliases for all GO- vulnerabilities.

amanning9 committed:
Triage and allowlist CVEs.

amanning9 committed:
Upgrade prometheus.prometheus to bring in CVE fixes.

amanning9 committed:
Bump versions of prometheus and grafana to mitigate vulns.

amanning9 committed:
Triage a load of go vulnerabilities.

amanning9 committed:
Triage final vuls from scan.

amanning9 committed:
Update nginx in guacamole to fix CVE.

m-bull committed:
Merge remote-tracking branch 'origin/fix-grype' into feat/noble

m-bull committed:
Merge branch 'main' into feat/noble

m-bull committed:
Merge branch 'main' into feat/noble

m-bull committed:
Install network-manager before desktop

m-bull committed:
Stop building Kubernetes 1.33

m-bull committed:
Switch to ed25519 keys in guacamole playbook

m-bull committed:
Update .grype.yaml to ignore unfixed Noble CVEs

m-bull committed:
Merge branch 'main' into feat/noble

m-bull committed:
Merge branch 'main' into feat/noble

mkjpryor committed:
Update prometheus to v3.13.1

m-bull committed:
Merge branch 'main' into feat/noble

m-bull committed:
CI: Remove has_ubuntu_builds

Building Ubuntu images is guarded in the check_paths job in pr.yml

m-bull committed:
Bump Noble baseimage to latest available

m-bull committed:
Split Grype config files by build name

Switch to individual Grype config files per build name, and use
grype-analyse tool to fail workflow runs.

m-bull committed:
Add complete grype config files for all images

m-bull committed:
Pull grafana from grafana/grafana

grafana/grafana-oss will not receive any more updates.

m-bull committed:
Bump image-builder to v0.1.55

m-bull committed:
Bump Nginx to 1.31.3

m-bull committed:
Add data to grype ignore list

m-bull committed:
Update RStudio to latest

m-bull committed:
Allow a suffix in dependency updates

m-bull committed:
Update Grype ignorelist

mkjpryor committed:
Update Kubernetes patch versions (v1.35.7, v1.36.3, v1.34.10)

m-bull committed:
Revert "TESTING ONLY: Use feature branch of az-ops in CI"

This reverts commit e37efa7b77ac7cf62e474a90168f0797b859a59f.

m-bull committed:
TEMP: Leave images on the cloud and in S3 for manual testing
